packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -278,14 +278,20 @@ export default abstract class Editor extends ObservableMixin() {
 		// Prefer the language passed as the argument to the constructor instead of the constructor's `defaultConfig`, if both are set.
 		const language = config.language || ( constructor.defaultConfig && constructor.defaultConfig.language );
 
-		this._context = config.context || new Context( { language } );
+		/**
+		 * We don't pass translations to the config, because its behavior of splitting keys
+		 * with dots (e.g. `resize.width` => `resize: { width }`) breaks the translations.
+		 */
```

Comment:
```suggestion
		// We don't pass translations to the config, because its behavior of splitting keys
		// with dots (e.g. `resize.width` => `resize: { width }`) breaks the translations.
```